README.md

```diff
@@ -36,16 +36,15 @@ We recommend using `yarn` because of [compatibility](https://github.com/c-hive/t
 </script>
 ```
 
-#### Config
+#### Configs
 
 <pre>
 <b>TeamContributionCalendar(container, gitHubUsers, gitLabUsers, proxyServerUrl)</b>
 </pre>
 
-- `container`: a DOM element in which the calendar will be rendered
-- `gitHubUsers` / `gitLabUsers`: array of user names
-- `proxyServerUrl`: CORS proxy url
-  - We serve one by default for _development purposes only_, no uptime guaranteed. Consider using your [own server](https://github.com/c-hive/cors-proxy). Keep it in mind, **otherwise you will most likely get 403 on production**.
```

Comment:
I don't prefer this list format to describe the options. How about a table similarly to https://github.com/c-hive/c-hive.dev?

Reply:
The other option I can think of is to include more examples.

Reply:
One thing I'm missing from the doc is that it's not clarified which config is optional or mandatory.

Reply:
Updated the readme https://github.com/c-hive/team-contribution-calendar/pull/64/commits/676a00f25716d8971ee44a6dcbb27f95f291e9b4